storer/storer.go

```diff
@@ -0,0 +1,15 @@
+package storer
+
+//Storer interface defines methods to manipulate ci-files including pipeline files and step logs
+type Storer interface {
+	GetName() string
+	GetLatestVersion(pipelinepath string) int
+	SavePipelineFile(pipelinePath string, content string) error
+	ReadLatestPipelineFile(pipePath string) (string, error)
+	SaveLogFile(pipelinePath string, stage string, step string, content string) error
+	ReadLogFile(pipelinePath string, stage string, step string) (string, error)
+}
```

Comment:
I will need a method to get specific version of pipeline file because we need to show history of it.
Then I think save log file should have version number too.